pyxem/signals/diffraction2d.py

```diff
@@ -582,6 +584,61 @@ def remove_background(self, method, **kwargs):
 
         return bg_subtracted
 
+    def as_polar(self, dr=1.0, dt=None, jacobian=False, *args, **kwargs):
+        """Reprojects two-dimensional diffraction data from cartesian to polar
+        coordinates.
+
+        Parameters
+        ----------
+        dr : float
+            Radial coordinate spacing for the grid interpolation
+            tests show that there is not much point in going below 0.5
+        dt : float
+            Angular coordinate spacing (in radians)
+            if ``dt=None``, dt will be set such that the number of theta values
```

Comment:
I'm a bit lost as to what this means. Is that because it's actually complicated or could we make the wording better here?